api/abortion_policy_api.py

```diff
@@ -192,3 +172,22 @@ def fill_in_missing_data(state_policies, defaults):
         for k, v in defaults.items():
             if k not in state_info.keys():
                 state_info[k] = v
+
+
+def to_json(data, file_names):
+    """
+    Dumps data to json file(s)
+
+    Inputs:
+        data (dict): list of dictionaries to output to JSON
+        file_names (list): list of file names to export to
+    """
+
+    print("Writing out to json file")
+    assert len(data) == len(
+        file_names
+    ), "Incorrect number of data dictionaries passed"
+
+    for i, file_name in enumerate(file_names):
+        with open(file_name, "w", encoding="utf-8") as f:
```

Comment:
Make a constant in the `utils.constants` file called `STANDARD_ENCODING` and use it where it's applicable.